content/material_and_questions.md

```diff
@@ -44,4 +53,153 @@ In the below table, any minimum or maximum variation between material years are
+
+Overtime Specifics
+Question Type maximums can be exceeded in Overtime.
+
+## Question Writing
+
+### Preparation of Questions
+All questions must contain verbatim material. On some types, the addition of a single, approved, interrogative word at the very beginning or very end is required.
+
+### Question Types
+
+#### Interrogative
+Should not be overly long and should be clear and precise.
+Must have exactly one inserted interrogative word.
+Must have a single answer.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+The Question Type abbreviation is "INT"
+
+#### Multiple Answer
+Must have exactly one inserted interrogative word.
+Must have more than one answer.
+- The clarification of a single answer does not meet the above requirement.
+- The identification of a singular pronoun does not meet the above requirement.
+- The identification of a plural pronoun does meet the above requirement.
+Must include a unique word, or 2-3 word unique phrase within the first 5 words of the question. 
+If the multiple answers contain at least one "positive" answer AND at least one "negative" answer, this is deemed invalid as a Multiple Answer question.
+The Question Type abbreviation is "MA"
+
+#### Reference
+Used to distinguish exact same words or phrases from the material.
+The entire question is part of the required question + answer.
+If a quizzer needs to provide the reference question, the provided reference question must: 
+- Be of the exact same type as the one on the card (CR, CVR, CRMA, CVRMA).  The quizmaster and/or answer judge must check to confirm this.
+- Have content AND wording is similar to the content and wording of the written question.
+    - For greater clarity, we have provided the following example: CVR According to Luke chapter 23: 37 King of what?  A. The Jews The quizzer can give the question “King of what?” or “King of whom?”, as Jews could be referring to either nation (what) or people (whom).  In this case the quizzer could also give “King of the what?”
+- Have the determining word or phrase from the quizmaster’s card. 
+    - In the 50/50 example, “What men?”, and “Lawless what?, the questions are not similar.
+The determination of a valid Reference question, and the valid sub-type only considers the content in the question. 
+
+##### Chapter-Verse
+Must have a single answer.
+The question must contain a word or phrase that exists more than once in the same chapter.
+Context is the single verse.
+The Question Type abbreviation is "CVR"
+
+##### Multiple-Answer, Chapter-Verse
+Must have more than one answer.
+The Question Type abbreviation is "MACVR"
+
+##### Chapter (only)
+The question must contain a word or phrase that exists more than once in the material.
+The Question Type abbreviation is "CR"
+
+##### Multiple-Answer, Chapter (only)
+Must have more than one answer.
```

Comment:
I would suggest adding an explicit statement here as with MACVRs above.

Reply:
Agreed.

Reply:
well, this will be covered by inheritance.  The question is how to reword the CR section.

Reply:
Right. I'd do something like:

> MACR is both a MA and a CR.

Reply:
Sure, but we need to deal with my latest comment on the MACVR section first. Whatever we do there I'll mirror here.